test/functorch/test_vmap.py

```diff
@@ -3708,11 +3708,6 @@ def test_vmap_exhaustive(self, device, dtype, op):
         # One or more of the overload doesn't have a Batch rule.
         xfail('where'),
         xfail('bincount'),
-        xfail('bitwise_and'),
-        xfail('bitwise_or'),
-        xfail('bitwise_xor'),
-        xfail('bitwise_left_shift'),
```

Comment:
I think for `bitwise_left_shift` and `bitwise_right_shift`, negative shifts are undefined (and IIRC few reference samples do produce negative shift).

Ref: https://github.com/pytorch/pytorch/issues/70904

Reply:
The behavior seems interesting. Currently, I xfailed the cpu tests.